lib/str.js

```diff
@@ -1127,4 +1118,5 @@ const Str = {
     },
 };
 
-export default Str;
+// eslint-disable-next-line import/prefer-default-export
+export { Str };
```

Comment:
Why not `export default Str`?

Reply:
This could lead to a lot of changes in other repos such as NewDot, apart from that the PR looks good!

Reply:
I don't know why but I thought I had an error if I didn't do it like this.

When retrying, I didn't get any anymore. It must have been when I was doing the refactoring, one edge case where I was in.

It's now corrected!